core/src/components/item/test/basic/item.e2e.ts

```diff
@@ -30,6 +30,10 @@ configs().forEach(({ title, screenshot, config }) => {
             <div slot="helper">Helper</div>
           </ion-item>
 
+          <ion-item-divider>
```

Comment:
Optional: I think it's confusing to add the item-divider to the item tests when item-divider has its own test directory in `item-divider/test`. I'll defer to you and Brandy though.

Reply:
My thinking was 

1. That it would be easier to understand what regression was occurring if Item & Item Divider are in the same test
2. It's more efficient to combine them into the same test
3. There are other tests in Item that include Item Dividers

but i can move them. It doesn't really matter. 🤷🏻